src/screens/points/PointsScreen.tsx

```diff
@@ -88,7 +88,8 @@ export default function PointsScreen() {
             initialRouteName={POINTS_ROUTES.CLAIM_CONTENT}
             screenOptions={{ swipeEnabled: false }}
             tabBarPosition="bottom"
-            tabBar={() => null}
```

Comment:
This is rendering the default tab bar on the points screen behind ours (see image below — I reduced the opacity of our tab bar so you can see it)

Maybe we can instead do something like this:

```
const EmptyTabBar = () => <></>;

export default function PointsScreen() {
  ...
  <Swipe.Navigator
      ...
      tabBar={EmptyTabBar}
  >
}
````

<img src="https://github.com/rainbow-me/rainbow/assets/7061887/57884c5b-2018-4024-b30a-7c07f2431e5d" width="400" />

Reply:
Does this still allow you to tab on the main navigation tab when using an Empty tab bar? My whole issue was with the tab bar set to null it didn't work anymore.